torch_np/tests/test_ndarray_methods.py

```diff
@@ -17,7 +17,7 @@ def test_indexing_simple(self):
 
         assert isinstance(a[0, 0], np.ndarray)
         assert isinstance(a[0, :], np.ndarray)
-        assert a[0, :].base is a
```

Comment:
Are these changes coming from a weird ordering of the stacks or what'st hte reason?

Reply:
Can remove it from this one if wanted.

Reply:
My point is, these are unrelated to this PR as, I believe, the `base` arg was removed in a different PR. As such, what are they doing here?